website/recursion_website/templates/base.html

```diff
@@ -0,0 +1,59 @@
+<!Doctype html>
+<html>
+<head>
+	<title>RECursion</title>
+	<!--Package Imports-->
+
+	<!-- Compiled and minified MATERIAL CSS -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <!-- Compiled and minified MATERIAL JavaScript -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+
+    <link rel="stylesheet" href="{% static 'css/base_css.css' %}">
+</head>
+<body>
+	<header>
+	<nav><!-- Navigation Color is controlled by class !-->
+		<div class="nav-wrapper #263238 blue-grey darken-3">
+			<a href="#" class="brand-logo"><img class="img" src="http://www.recursionnitd.in/images/log.png"></a>
+			<ul id="nav-mobile" class="right hide-on-med-and-down">
+				<li><a href="/events.html">Events</a></li>
+				<li><a href="/askREC.html">AskREC</a></li>
+				<li><a href="/start.html">Getting Started</a></li>
+				<li><a href="/login.html">Login</a></li>
+			</ul>
+		</div>
+	</nav>
+	</header>
+	<main>
+	{% block content %}
+	{% endblock content %}
+	</main>
+	<footer class="page-footer #263238 blue-grey darken-4"><!-- Footer Color is controlled by class !-->
+		<div class="container">
+			<div class="row">
+				<div class="col 16 s6">
+					<h5 class="white-text">Footer Content</h5>
+                	<p class="grey-text text-lighten-4">You can use rows and columns here to organize your footer content.</p>

```

Comment:
"You can use rows and columns here to organize your footer content." Will this text be displayed to the users? What am I missing here?

Reply:
No bhaiya, It will be updated, actually i wrote sample text just to have an idea of how text will look.

Reply:
(y)